src/components/footer/Footer.js

```diff
@@ -1,24 +0,0 @@
-import "./style.css";
-import instagram from "./../../img/icons/instagram.svg";
-import gitHub from "./../../img/icons/gitHub.svg";
-import twitter from "./../../img/icons/twitter.svg";
-import linkedIn from "./../../img/icons/linkedIn.svg";
-const Footer = () => {
-    return (<footer className="footer">
-    <div className="container">
-        <div className="footer__wrapper">
-            <ul className="social">
-                <li className="social__item"><a href="#!"><img src={instagram} alt="Link"/></a></li>
```

Comment:
use iteration over array instead of hardcoding all "li" elements
How to: https://medium.com/how-to-react/different-ways-to-loop-through-arrays-and-objects-in-react-39bcd870ccf